### PR TITLE
(Chore) Remove canvas to fix Netlify build

### PR DESCRIPTION
**Pull request by MauritsR**
Removes canvas from package.json so that it's dependencies don't have to be installed (Dockerfile / ci)

This should also fix the Netlify build, which was not updated to perform the above installations.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -29,9 +29,8 @@ jobs:
           path: ~/.npm
           key: ${{ runner.os }}-build-${{ env.cache-name }}
 
-      # See: https://github.com/Automattic/node-canvas#compiling
       - name: Install required libraries
-        run: sudo apt-get install build-essential libcairo2-dev libpango1.0-dev libjpeg-dev libgif-dev librsvg2-dev
+        run: sudo apt-get install libgif-dev
```
Comment (jonkoops) [inline .github/workflows/ci.yml:33]: What is `libgif-dev` needed for?

**Comment by MauritsR** [comment]
Closed because the PR caused unit tests to fail on CI - checking it out

**Comment by jonkoops** [comment]
@MauritsR FYI the author of `canvas` just published the binaries for Node 16 on Linux and macOS (Windows still missing). See: https://github.com/Automattic/node-canvas/issues/1791#issuecomment-829491829

**Comment by MauritsR** [comment]
> @MauritsR FYI the author of `canvas` just published the binaries for Node 16 on Linux and macOS (Windows still missing). See: [Automattic/node-canvas#1791 (comment)](https://github.com/Automattic/node-canvas/issues/1791#issuecomment-829491829)

👍 Was already looking at it - happy to see the binaries added so fast. Netlify now builds correctly (would have been quite a hassle to install the binaries there). 

Closing this PR but will open a new one to remove redundant binaries install.

**Pull request closed without merging**